Makefile

```diff
@@ -8,9 +8,25 @@ PROJECT_URL := "https://github.com/scootdev/scoot"
 default:
 	go build ./...
 
+dependencies: 
+	# get all dependencies needed to run scoot / check if there have
+	# been updates since they were first installed
+	go get -t -u github.com/scootdev/scoot/...
+
+	# mockgen is only referenced for code gen, not imported directly
+	go get github.com/golang/mock/mockgen
```

Comment:
ran a little experiment, deleted my whole bin directory, and then ran:

- make dependencies
-make testlocal

everything passes and bin/mockgen is created in the process so I don't think we need to explicitly install this.